### PR TITLE
Split advanced.md 

### LINKED ISSUE #95: Split 'Advanced Usage' doc into separate docs
The Micros docs can have three separate pages, replacing the 'advanced usage' page:

- API Keys
- Visor
- Technical Details 

Each page could have a detailed doc / tutorial with context as to 'why' each piece is useful.

### PR DESCRIPTION
**Pull request by Ninja243**
`advanced.md` has now been split into three files, `visor.md`, `api_keys.md` and `details.md`. The visor page now walks the user through using Visor.

Closes https://github.com/deta/docs/issues/95

### CODE REVIEW

**Comment by xeust** [comment]
Great start!

A couple notes of feedback.

For me as a reader, the connection between the descriptive words in the doc and using the actual product is a bit abstract.

## Generally

Ideally as a reader I have:

1. context as to *why* the given feature exists / a connection to the larger product and end user objective
2. a visual description / example / code snippet that helps me understand *exactly* how to operate any given feature, once I have identified I am in, or have been in, the 'why' situation from 1. The more concrete / visual, the better.

### Example 1: with respect to the preset environment variables

The context as to why these env vars exist feels missing (using this information in the code).

One example of a user context that illustrates a why:

Let's say I'm using an `express` node server. I want to boot up the server locally on port 3000, but this step is unnecessary on Deta. Nonetheless, I want the same code base. To solve this problem, I can create a conditional.

- In this case the `env` var `DETA_RUNTIME` is relevant to creating a conditional. So a code snippet in context could be super helpful:

```js
if (!process.env.DETA_RUNTIME) {
    app.listen(port, ...)
}
```

### Example 2: with respect to the Visor docs

Ideally each feature would be shown / described in context. 

What is the connection of each piece to the larger product feature (real time debugging) & how do I use it? (e.g. what does the 'pencil' button do).

On this note, a product tour with pictures for Visor (as Base UI does) could be awesome, imo!
- https://blog.deta.sh/posts/base_guide/
- https://docs.deta.sh/docs/base/base_ui

Also the video is great, I wonder if we can embed the YouTube video at the top (i.e. playable in the doc?)

___
Role-model docs, imo:
- https://docs.djangoproject.com/en/3.2/intro/overview/
- https://stripe.com/docs

Inverse role-model docs, imo:
- https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/appconfig.html#paginators

File: docs/micros/visor.md [hunk 1]
```diff
@@ -1,8 +1,52 @@
...
+    values={[
+    { label: 'Browser', value: 'browser' },
+    { label: 'Terminal', value: 'terminal' },
+    ]
+}>
+<TabItem value="browser">
+
+Start off by logging into your console at [deta.sh](https://deta.sh).
+
+Once that's done, choose a micro to view by clicking on itname (under the _"Micros"_ tab).
```
Comment (xeust) [inline docs/micros/visor.md]: `Once that's done, choose a micro to view by clicking on its name (under the _"Micros"_ tab).`

File: docs/micros/visor.md [hunk 2]
```diff
@@ -1,8 +1,52 @@
...
+
+Now click on _"Visor"_
+
+</TabItem>
+<TabItem value="terminal">
+
+Start your terminal in the root directory of your deployed Detproject.
```
Comment (xeust) [inline docs/micros/visor.md]: `Start your terminal in the root directory of your deployed Deta project.`

File: docs/micros/visor.md [hunk 3]
```diff
@@ -1,8 +1,52 @@
...
+Run `deta visor open`.
+
+</TabItem>
+</Tabs>
+
+### Using Visor
+When viewing the Visor page, you'll see a link to your Micro in the top right, a button named _"HTTP Client"_ in the bottom left and a list of handled events by your Micro!
+
+#### The Event List
+Every event has the returned status code, the path, the protocol, the method and the time logged above it. Under this information are three  tabs, letting you view the response given by the Micro, the Request sent to the Micro and logs (normally displaying errors if they were to occur).
```
Comment (xeust) [inline docs/micros/visor.md]: We actually have four tabs, 'Errors' being the last one. 

I personally think playing around with all the edge cases & documenting the product with visuals in the process would be a great addition here!

File: docs/micros/visor.md (hunk 1 above)
Comment (Ninja243) [reply docs/micros/visor.md]: Good spot, thanks!

File: docs/micros/visor.md (hunk 3 above)
Comment (Ninja243) [reply docs/micros/visor.md]: We do? :D

grafik


Good idea on the visuals and edge case idea, I'll make sure to work those in!

**Pull request closed without merging**